src/components/Layout/Header.tsx

```diff
@@ -24,11 +27,109 @@ const onRightClickLogo = (event: React.MouseEvent): void => {
     }
 }
 
+const navLinks = [
+    {
+        section: 'Product',
+        items: [
+            {
+                title: 'Code Search',
+                href: '/code-search',
+            },
+            {
+                title: 'Batch Changes',
+                href: '/batch-changes',
+            },
+            {
+                title: 'Code Insights',
+                href: '/code-insights',
+            },
+            {
+                title: 'Code Intelligence',
+                href: 'https://docs.sourcegraph.com/code_intelligence',
+            },
+        ],
+    },
+    {
+        section: 'Resources',
+        items: [
+            {
+                title: 'Blog',
+                href: '/blog',
+            },
+            {
+                title: 'Learn',
+                href: 'https://learn.sourcegraph.com/',
+            },
+            {
+                title: 'Dev Tool Time',
+                href: 'https://www.youtube.com/playlist?list=PL6zLuuRVa1_iDEP4EicZ8972RgyccCRGF',
+            },
+            {
+                title: 'Sourcegraph Podcast',
+                href: '/podcast',
+            },
+            {
+                title: 'Case Studies',
+                href: '/case-studies',
```

Comment:
Please note we have moved the Case Studies link into the Resources dropdown per stakeholder feedback. It has been removed from the Use Cases dropdown.